packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -10,27 +10,77 @@ import 'box.dart';
 import 'sliver.dart';
 import 'sliver_multi_box_adaptor.dart';
 
+/// A sliver that contains multiple box children that have the same extent in
+/// the main axis.
+///
+/// [RenderSliverFixedExtentBoxAdaptor] places its children in a linear array
+/// along the main axis. Each child is forced to have the [itemExtent] in the
+/// main axis and the [SliverConstraints.crossAxisExtent] in the cross axis.
```

Comment:
You should mention here that `itemExtent` isn't just a field here but is something you override in a subclass.